config-templates/shopify/order.yaml

```diff
@@ -0,0 +1,59 @@
+data_source:
+  name: shopify_orders
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY."ORDER" ### Update to your opportunities schema.table
+  mutability:
+    type: immutable
+  identifiers:
+    - name: order
+      type: primary
+      expr: id
+    - name: shipping_address_id
```

Comment:
most of these foreign keys don't have matching primary keys in other datasources - I could see an argument for still including them, but I think it may be confusing. Open to discussing it!